packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -233,7 +233,11 @@ export class PendingStateManager implements IDisposable {
 
             // then we push onto pendingStates which will cause PendingStateManager to resubmit when we connect
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            this.pendingStates.push(this.initialStates.shift()!);
+            const message = this.initialStates.shift()!;
+            this.pendingStates.push(message);
+            if (message.type === "message") {
```

Comment:
#nit technically `message` is a pending state which may be of type message, so the const would probably be better named as `firstPendingState` or something (naming is hard)

Reply:
Fixing